curriculum/challenges/english/25-front-end-development/workshop-library-manager/67116d7584d0b469b14579c3.md

```diff
@@ -30,7 +30,7 @@ assert.lengthOf(getBooksByAuthor, 2)
 Your `getBooksByAuthor` function should return an array.
 
 ```js
-assert.isArray(getBooksByAuthor(library))
+assert.isArray(getBooksByAuthor(library, "Arvid Kahl"));
```

Comment:
```suggestion
assert.isArray(getBooksByAuthor(library, ''));
```
Perhaps empty string might be even better? While filtering with existing author is tested in last test, there isn't any test checking what is returned if there's no books matching.